students/Jabalt/Assignment1/fibonacci.py

```diff
@@ -0,0 +1,11 @@
+def fibonacci(n):
+    if n == 1:
+#the first fibonacci nubmer is 1 
+        return 1
+    elif n ==2 :
+        return 1
+    elif n >2 :
+        return fibonacci(n-1) + fibonacci(n-2)
+for n in range (1,11):
+     print (fibonacci(n))
+        
```

Comment:
Lucas numbers are work in much the same way as fibonacci, you just need to change your 0 and 1 to deal with 1, and 2.